vaadinfrontend/frontend/themes/datamanager/components/spreadsheet.css

```diff
@@ -0,0 +1,21 @@
+vaadin-combo-box-overlay::part(content) {
+  height: 30rem;
+  width: 20rem;
+  max-height: 30vh;
+}
+
+vaadin-combo-box-overlay::part(overlay) {
+  height: 30rem;
+  width: 20rem;
+  max-height: 30vh;
+}
```

Comment:
This is dangerous! you are styling a vaadin component here not only for spreadsheets but globally. Should go into custom.css directly or be more specific imho.